src/web/client/extension.ts

```diff
@@ -111,6 +108,11 @@ export function activate(context: vscode.ExtensionContext): void {
                     queryParamsMap,
                     context.extensionUri
                 );
+
+                const orgId = queryParamsMap.get(queryParameters.ORG_ID) as string;
```

Comment:
Could you please remove orgURL logging in check params API

Reply:
Done!